docs/develop/01_sdk/02_cookbook/06_upgrading_to_v0_29/01_backwards_compatibility.md

```diff
@@ -0,0 +1,54 @@
+---
+id: v29-backwards-compatibility
+title: Backwards Compatibility / Interoperability with Previous Versions
+---
+
+Beside the breaking code changes we have identified two data format changes which might affect interoperability.
+The didsign.io and w3n.id already accept bare requests for attestation, which are effectively the new Credentials, so nothing should break there.
+
+In the attester flow the `request-attestation` message is affected.
+In the verifier flow the `submit-credential` message is affected.
```

Comment:
I now vaguely remember Timo saying that yet another message will be affected, probably by the CType schema change